plugins/module_utils/_api_data.py

```diff
@@ -13,6 +13,7 @@
 class APIData(object):
     def __init__(self, primary_keys=None,
                  stratify_keys=None,
+                 require_one_of=None,
```

Comment:
Sorry for not noticing this earlier, but can you rename this to `required_one_of` (with `d`)? That's the same name used by ansible-core's argument_spec.

Reply:
Done.